pkg/controller/controller.go

```diff
@@ -41,6 +41,13 @@ func NewManager(ctx context.Context, config *config.Config, cfg *rest.Config, op
 		return nil, err
 	}
 
+	if config.KubeAPIOptions.Burst > 0 {
+		mgr.GetConfig().Burst = config.KubeAPIOptions.Burst
+	}
+	if config.KubeAPIOptions.QPS > 0 {
+		mgr.GetConfig().QPS = float32(config.KubeAPIOptions.QPS)
```

Comment:
Because the field is defined as a float32 in client-go, see https://pkg.go.dev/k8s.io/client-go/rest#Config.QPS. I just did not see why would need it that fine-grained. Tekton also exposes it as an int configuration parameter.

Reply:
oh, I see.